go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/containerd/containerd/v2
 
-go 1.21
+go 1.22
```

Comment:
Looks like now this has to be `go X.Y.Z`, not `go X.Y`

https://github.com/containerd/containerd/actions/runs/8793507331/job/24131505725?pr=10110

```
Run actions/setup-go@v5
Setup go version spec 1.21.9
Found in cache @ /opt/hostedtoolcache/go/1.21.9/x64
Added go to the path
Successfully set up Go version 1.21.9
go: downloading go1.22 (linux/amd64)
go: download go1.22 for linux/amd64: toolchain not available
Error: Command failed: go env GOPATH
go: downloading go1.22 (linux/amd64)
go: download go1.22 for linux/amd64: toolchain not available
```

Reply:
Yes, otherwise a `toolchain` directive will be added.